src/applications/vaos/new-appointment/index.jsx

```diff
@@ -73,200 +66,116 @@ export function NewAppointment() {
-  }
   return (
-    <FormLayout pageTitle={crumb}>
+    <FormLayout pageTitle={pageTitle}>
       <Switch>
-        <Route path={`${match.url}/contact-info`} component={ContactInfoPage} />
-        <Route
-          path={`${match.url}/choose-facility-type`}
-          component={TypeOfFacilityPage}
-        />
-        <Route
-          path={`${match.url}/choose-visit-type`}
-          component={TypeOfVisitPage}
-        />
-        <Route
-          path={`${match.url}/choose-sleep-care`}
-          component={TypeOfSleepCarePage}
-        />
-        <Route
-          path={`${match.url}/choose-eye-care`}
-          component={TypeOfEyeCarePage}
-        />
-        <Route
-          path={`${match.url}/audiology`}
-          component={TypeOfAudiologyCarePage}
-        />
-        <Route
-          path={`${match.url}/preferred-date`}
-          component={PreferredDatePage}
-        />
         <Route
-          path={`${match.url}/request-date`}
-          component={DateTimeRequestPage}
-        />
-        <Route
-          path={`${match.url}/select-date`}
-          component={DateTimeSelectPage}
-        />
+          path={[
+            `${match.url}/va-request/contact-information`,
+            `${match.url}/community-request/contact-information`,
+            `${match.url}/contact-information`,
+          ]}
+        >
+          <ContactInfoPage c />
+        </Route>
+        <Route path={`${match.url}/facility-type`}>
+          <TypeOfFacilityPage />
+        </Route>
         <Route
-          path={`${match.url}/va-facility-2`}
-          component={VAFacilityPageV2}
-        />
+          path={[
+            `${match.url}/va-request/preferred-method`,
+            `${match.url}/choose-visit-type`,
+          ]}
+        >
+          <TypeOfVisitPage />
+        </Route>
+        <Route path={`${match.url}/sleep-care`}>
+          <TypeOfSleepCarePage />
+        </Route>
+        <Route path={`${match.url}/eye-care`}>
+          <TypeOfEyeCarePage />
+        </Route>
+        <Route path={`${match.url}/audiology-care`}>
+          <TypeOfAudiologyCarePage />
+        </Route>
+        <Route path={`${match.url}/preferred-date`}>
+          {featureUseVaDate ? (
+            <PreferredDatePageVaDate />
+          ) : (
+            <PreferredDatePage />
+          )}
+        </Route>
+        <Route path={`${match.url}/date-time`}>
+          <DateTimeSelectPage />
+        </Route>
+        <Route exact path={`${match.url}/va-request/`}>
+          <VARequest />
+        </Route>
+        <Route exact path={`${match.url}/community-request/`}>
+          <CCRequest />
+        </Route>
+        <Route path={`${match.url}/location`}>
+          <VAFacilityPageV2 />
+        </Route>
+        <Route path={`${match.url}/provider`}>
+          <ProviderSelectPage />
+        </Route>
         <Route
           path={`${match.url}/how-to-schedule`}
           component={ScheduleCernerPage}
         />
         <Route
```

Comment:
While self closing `<Route />`  tag is completely valid, let's keep things consistent by using the opening and closing `<Route></Route>` like the other patterns

Reply:
@ryanshaw, Can you change the above lines to match below? Everything else looks terrific!

```
        <Route path={`${match.url}/how-to-schedule`}>
          <ScheduleCernerPage />
        </Route>
```